pkg/scheduler/framework/plugins/nodeaffinity/node_affinity_test.go

```diff
@@ -790,6 +791,52 @@ func TestNodeAffinity(t *testing.T) {
 			name:       "Doesn't match added affinity",
 			wantStatus: framework.NewStatus(framework.UnschedulableAndUnresolvable, errReasonEnforced),
 		},
+		{
+			pod: &v1.Pod{
+				Spec: v1.PodSpec{
+					NodeSelector: map[string]string{
+						"foo": "bar",
+					},
+				},
+			},
+			labels: map[string]string{
+				"foo": "bar",
+				"baz": "blah",
+			},
+			name:             "Matches node selector correctly even if PreFilter is not called",
+			disablePreFilter: true,
+		},
+		{
+			pod: &v1.Pod{
+				Spec: v1.PodSpec{
+					Affinity: &v1.Affinity{
+						NodeAffinity: &v1.NodeAffinity{
+							RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+								NodeSelectorTerms: []v1.NodeSelectorTerm{
+									{
+										MatchExpressions: []v1.NodeSelectorRequirement{
+											{
+												Key:      "GPU",
+												Operator: v1.NodeSelectorOpExists,
+											}, {
+												Key:      "GPU",
+												Operator: v1.NodeSelectorOpNotIn,
+												Values:   []string{"AMD", "INTER"},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			labels: map[string]string{
+				"GPU": "NVIDIA-GRID-K1",
+			},
+			name:             "Matches node affinity correctly even if PreFilter is not called",
```

Comment:
Let's move the name of the test to the beginning of the struct. Please fix the existing tests too.